projects/demo/src/modules/pipes/filter/examples/1/component.ts

```diff
@@ -29,5 +29,5 @@ export class TuiFilterExample1 {
         },
     ];
 
-    readonly matcher = (item: Item, search: number): boolean => item.price > search;
+    readonly matcher = (item: DemoItem, search: number): boolean => item.price > search;
```

Comment:
can we just exclude `demo`-app from this rule ?

Reply:
all demo-examples should be as simple as possible.

I am not sure that this prefix `Demo` makes example better.